src/config/syscheck-config.c

```diff
@@ -2573,12 +2512,20 @@ static int process_option_regex(char *option, OSMatch ***syscheck_option, xml_no
 static void process_option(char ***syscheck_option, xml_node *node) {
 
     unsigned int counter_opt = 0;
+    char *real_path;
+    char *dir = strdup(node->content);
+    char *tmp_dir;
     char **new_opt = NULL;
 
+    tmp_dir = validate_path(dir);
+    if (!tmp_dir) {
+        return;
```

Comment:
Variable `dir` is not freed
```suggestion
                os_free(dir);
                return;
```